concourse/pipelines/replicator.py

```diff
@@ -345,13 +353,40 @@ def process_results(self, results):
                 info('removing pipeline: {p}'.format(p=pipeline_name))
                 concourse_api.delete_pipeline(pipeline_name)
 
+            # trigger resource checks for new pipelines
```

Comment:
how about moving this to a method (_initialise_new_pipeline_resources of sth. similar)